### PR TITLE
Adding Pokemon Search feature.

### PR DESCRIPTION
**Pull request by chug2k**
I wanted to find Blastoise; so I added a search feature.

New method called `handleSearchText` that filters through `this.state.pokemon`.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -89,13 +90,22 @@ class App extends React.Component {
...
             <Form inline>
-              <FormControl type="text" placeholder="Search" className="mr-sm-2" />
+              <FormControl value={this.state.searchText}  onChange={(e) => this.handleSearchChange(e)} type="text" placeholder="Search" className="mr-sm-2" />
               <Button variant="outline-success">Search</Button>
             </Form>
           </Navbar.Collapse>
         </Navbar>
     )
   }
+  handleSearchChange = (event) => {
```
Comment (chug2k) [inline src/App.js:100]: Wait can you double check that this works with the filter by category feature?

File: src/App.js [hunk 2]
```diff
@@ -89,13 +90,22 @@ class App extends React.Component {
               </NavDropdown>
             </Nav>
             <Form inline>
-              <FormControl type="text" placeholder="Search" className="mr-sm-2" />
+              <FormControl value={this.state.searchText}  onChange={(e) => this.handleSearchChange(e)} type="text" placeholder="Search" className="mr-sm-2" />
```
Comment (PrimeTimeTran) [inline src/App.js:93]: If we made it an anonymous function on line 100, do we need to it again on this onChange?

File: src/App.js (hunk 1 above)
Comment (PrimeTimeTran) [reply src/App.js:100]: Ok I'm testing now.

File: src/App.js (hunk 2 above)
Comment (chug2k) [reply src/App.js:93]: That's a great question, Loi! I think we need to do it here, because this is a separate function than the one on line 100.

**Pull request merged**